pkg/storage/config.go

```diff
@@ -2,25 +2,26 @@ package storage
 
 import (
 	"context"
-	"database/sql"
 	"errors"
 	"os"
 
 	"github.com/golang-migrate/migrate/v4"
-	"github.com/golang-migrate/migrate/v4/database/postgres"
 	_ "github.com/golang-migrate/migrate/v4/source/file" // used internally by migrate
 	_ "github.com/lib/pq"                                // must remain here for sql lib to find the postgres driver
+)
+
+type connectionType int
 
-	"github.com/asecurityteam/asset-inventory-api/pkg/domain"
+// Database Connection types
+const (
+	Primary connectionType = iota
+	Replica
 )
 
 // PostgresConfig contains the Postgres database configuration arguments
 type PostgresConfig struct {
-	Hostname         string
-	Port             uint16
-	Username         string
-	Password         string
-	DatabaseName     string
+	URL              string
+	ReplicaURL       string
```

Comment:
do we need to add the replica url as an env var?

Reply:
Yes, but it is optional now. I.e. the code falls back to using primary for everything is the replica url is not set or is not valid/reachable.